docs: add information about Griot

CBID2 opened the pull request:
## Description
I noticed that there was no category for Caribbean food, so I created one and added information about Griot . 
<!--
Please do not leave this blank.
This PR [adds/removes/fixes/replaces] the [feature/bug/etc].
-->

## What type of PR is this? (check all applicable)

- [ ] ☝️ Add a pizza fact or trivia
- [ ] 🧑‍🍳 Add a pizza recipe
- [ ] 🗺️ Add a regional pizza
- [x] 🍴 Add a traditional food
- [ ] 📝 Documentation Update

## Related Tickets & Documents
Related to #86 
<!--
Please use this format link issue numbers: Fixes/Relates to #123
https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
-->

## Added to documentation?

- [ ] 📜 README.md
- [x] 🙅 no documentation needed

## [optional] What GIF best describes this PR or how it makes you feel?

<!-- note: PRs with deleted sections will be marked invalid. -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.

  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

adiati98 reviewed (changes requested):
Hey @CBID2,

Thank you for this addition! ✨ 

As I'm not too familiar in which continent Caribbean is located, I googled it. Most resources — I link to some of them below for your reference — say that it's part of North America.

- https://en.wikipedia.org/wiki/List_of_sovereign_states_and_dependent_territories_in_North_America
- https://visaguide.world/north-america-caribbean/
- https://www.quora.com/Is-the-Caribbean-a-part-of-North-or-South-America

That said, can you please list your addition under North America?

Thanks. 🙂

CBID2 commented:
> Hey @CBID2,
> 
> 
> 
> Thank you for this addition! ✨ 
> 
> 
> 
> As I'm not too familiar in which continent Caribbean is located, I googled it. Most resources — I link to some of them below for your reference — say that it's part of North America.
> 
> 
> 
> - https://en.wikipedia.org/wiki/List_of_sovereign_states_and_dependent_territories_in_North_America
> 
> - https://visaguide.world/north-america-caribbean/
> 
> - https://www.quora.com/Is-the-Caribbean-a-part-of-North-or-South-America
> 
> 
> 
> That said, can you please list your addition under North America?
> 
> 
> 
> Thanks. 🙂 

Made the change @adiati98! 😊

adiati98 commented on world-traditional-food.md:
```diff
@@ -19,18 +13,34 @@ If your hometown or country has a traditional food (especially if it's your favo
 
+- [Table of Contents](#table-of-contents)
 - [North America](#north-america)
+  - [Caribbean](#caribbean)
```
Because we want to list the countries, let's remove the Caribbean here.

adiati98 commented on world-traditional-food.md:
```diff
@@ -19,18 +13,34 @@ If your hometown or country has a traditional food (especially if it's your favo
 
 ## North America
 
+### Caribbean
```
Let's remove this as well.

adiati98 reviewed (changes requested):
Hey @CBID2,

I've left a couple of reviews here.

Also, we've merged a PR before yours. So, please resolve the conflicts before we can review your PR. Thanks! ✨

CBID2 commented:
Hey @adiati98. I made the changes

adiati98 commented on world-traditional-food.md:
```diff
@@ -1,9 +1,3 @@
-# Traditional Food Around the World
-
-Welcome to the Traditional Food Around the World section of our repo! 🗺️🍴
-
-If your hometown or country has a traditional food (especially if it's your favorite dish!), we encourage you to share it here. We're excited to learn about food culture from around the world!
-
```
I'm not sure why you deleted these. We don't want to remove them. So, please add them back.

adiati98 commented on world-traditional-food.md:
```diff
@@ -19,18 +13,31 @@ If your hometown or country has a traditional food (especially if it's your favo
+
+Griot (pronounced "gree-oh") is a popular Haitian dish. It is pork marinated in a mixture of herbs and spices and fried in oil, resulting in a crispy and tender meat, and the sauce is rich and flavorful. Griot is often topped with a pickled vegetable sauce called "pikliz"(pronounced "pick-leez") and served along side with fried plantains, rice, and red beans, making it the national dish of Haiti.
+
+ <img src="./images/griot.jpg" width="400" alt="a plate of rice and beans with fried plantains and griot topped with pikliz">
 ## South America
```
Add a space here.

```suggestion
 <img src="./images/griot.jpg" width="400" alt="a plate of rice and beans with fried plantains and griot topped with pikliz">
 
## South America
```

adiati98 reviewed (changes requested):
@CBID2 there are things to fix here.

Also, can you update your branch with the `main` branch?
We've merged a PR that added Indian Samosa, but I don't see it here.

Thanks! ✨

CBID2 commented:
> @CBID2 there are things to fix here.
> 
> Also, can you update your branch with the `main` branch? We've merged a PR that added Indian Samosa, but I don't see it here.
> 
> Thanks! ✨

Hey @adiati98. I made the changes.

adiati98 reviewed (changes requested):
Hey @CBID2,

Can you please ask for review once you've addressed all feedback?
I see you quoted my comment, but you haven't addressed something there.

Thanks.

CBID2 commented:
> Hey @CBID2,
> 
> Can you please ask for review once you've addressed all feedback? I see you quoted my comment, but you haven't addressed something there.
> 
> Thanks.

Hey @adiati98. I reread your previous feedback. The information for the Samosa is on line 61. For some odd reason, GitHub won’t let me link it here.

adiati98 commented:
@CBID2, have you tried to pull it locally?

CBID2 commented:
> @CBID2, have you tried to pull it locally?

Yes @adiati98

adiati98 reviewed (commented):
Hey @CBID2,

Thanks for the changes! ✨ 

FYI, I've removed some things as below:

- The foods that you've added to the Table of Contents in [this commit](https://github.com/open-sauced/pizza-verse/pull/100/commits/8023682ab9e9d380d41cedd56a90a4d60bad200d).
-  Duplication of image resource that somehow got added in [this commit](https://github.com/open-sauced/pizza-verse/pull/100/commits/31ee34ed7cff75a585ed62408c57aca3007cd511).

![go time gif](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExb2huMmxkNmYybWlkN3Z6NG9yeTNnanhlcDNzZWp5M2R3cHZpNWcxOCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/fyPFP2It34cgG87ds6/giphy.gif)

CBID2 commented:
Yay! Now everybody gets to enjoy the awesomeness of Haitian food @adiati98! 😊
